spec.html

```diff
@@ -19002,7 +19014,9 @@ <h1>Runtime Semantics: DefineMethod</h1>
         1. Else,
           1. Let _kind_ be ~Method~.
           1. Let _prototype_ be %Function.prototype%.
-        1. Let _closure_ be FunctionCreate(_kind_, |UniqueFormalParameters|, |FunctionBody|, _scope_, _prototype_).
+        1. If _name_ is not present as a parameter, then
```

Comment:
```suggestion
        1. If _name_ is not present, then
```
The convention is just "is present" and "is not present".